src/coreclr/jit/gentree.h

```diff
@@ -6085,15 +6089,66 @@ struct GenTreeJitIntrinsic : public GenTreeMultiOp
     NamedIntrinsic gtHWIntrinsicId;
 
 public:
-    regNumber GetOtherReg() const
+    //-----------------------------------------------------------
+    // GetRegNumByIdx: Get regNumber of i'th position.
+    //
+    // Arguments:
+    //    idx   -   register position.
+    //
+    // Return Value:
+    //    Returns regNumber assigned to i'th position.
+    //
+    regNumber GetRegNumByIdx(unsigned idx) const
```

Comment:
Should this (and the Set function) be inline? Or do we rely of the compiler to make that choice?

Reply:
For other definitions of sibling classes, for this methog, we have usually relied on compiler to make that choice. So will keep it as is.